packages/frontend-core/src/components/grid/controls/ColumnsSettingButton.svelte

```diff
@@ -2,16 +2,29 @@
   import { getContext } from "svelte"
   import { ActionButton, Popover } from "@budibase/bbui"
   import ColumnsSettingContent from "./ColumnsSettingContent.svelte"
+  import { FieldPermissions } from "../../../constants"
```

Comment:
Should we move these to `@budibase/types`?

Reply:
No, they are only used in the frontend. The API expects the `hidden` and `readonly` fields, mapped accordingly on the frontend